test/e2e/framework/util.go

```diff
@@ -153,80 +152,29 @@ func getServicesProxyRequest(c *client.Client, request *restclient.Request) (*re
 	return request.Prefix("proxy").Resource("services"), nil
 }
 
-func GetServicesProxyRequest(c *client.Client, request *restclient.Request) (*restclient.Request, error) {
```

Comment:
I deleted this since it is just exporting the above function, which has an identical API. Instead, I just exported the above function.